main/settings_celery.py

```diff
@@ -6,6 +6,7 @@
 
 from main.envs import get_bool, get_int, get_string
 
+DEV_ENV = get_bool("DEV_ENV", False)  # noqa: FBT003
```

Comment:
Could use `settings.ENVIRONMENT` instead of creating this new one

Default value locally is "dev"

```python
    if settings.ENVIRONMENT == "dev":
        // do something
```